src/a2acallers.psm1

```diff
@@ -54,7 +54,7 @@ function Invoke-SafeguardA2aMethodWithCertificate
 
     $Service = $Service.ToLower()
 
-    $local:BodyInternal = $null
+    $local:BodyInternal = ""
```

Comment:
Are you sure that you are working from the latest master.  I fixed this same issue last month and the code comparison here doesn't look like the code in master.  Here is the PR where this should already be fixed.  https://github.com/OneIdentity/safeguard-ps/pull/484

Reply:
you're right.  my fault - i based my branch on my fork's master instead of the OID master.  Canceling the PR in favor of what's already fixed.